airbyte-webapp/src/views/Connection/CatalogTree/CatalogTree.tsx

```diff
@@ -1,59 +1,65 @@
-import { Field, FieldProps, setIn, useFormikContext } from "formik";
-import React, { useCallback } from "react";
+import React, { useCallback, useMemo, useState } from "react";
+
+import { LoadingBackdrop } from "components/ui/LoadingBackdrop";
 
 import { SyncSchemaStream } from "core/domain/catalog";
-import { AirbyteStreamConfiguration } from "core/request/AirbyteClient";
+import { BatchEditProvider } from "hooks/services/BulkEdit/BulkEditService";
```

Comment:
It feels like we're using `Batch` and `Bulk` interchangeably in this service.  Maybe not a fix for this PR though.

Reply:
As I was working on this more, I noticed this as well. Updated!